Source/WebCore/platform/graphics/displaylists/DisplayListItems.cpp

```diff
@@ -276,15 +280,20 @@ void DrawFilteredImageBuffer::dump(TextStream& ts, OptionSet<AsTextFlag> flags)
 
 void DrawGlyphs::apply(GraphicsContext& context) const
 {
+#if USE(SKIA)
+    if (m_textBlob)
+        static_cast<GraphicsContextSkia*>(&context)->drawSkiaText(m_textBlob, SkFloatToScalar(m_localAnchor.x()), SkFloatToScalar(m_localAnchor.y()), m_enableAntialiasing, m_isVertical);
```

Comment:
Hmm, what about other types of GraphicsContexts also used by our ports? e.g. NullGraphicsContext used as GraphicsContext? I think this is error-prone. Why not introduce a new cross-platform virtual method, that only our ports implement?